mock/src/test/java/com/tngtech/keycloakmock/impl/UrlConfigurationTest.java

```diff
@@ -28,7 +30,13 @@ private static Stream<Arguments> server_config_and_expected_base_url() {
         Arguments.of(aServerConfig().withPort(80).build(), "http://localhost"),
         Arguments.of(aServerConfig().withPort(443).build(), "http://localhost:443"),
         Arguments.of(aServerConfig().withTls(true).withPort(80).build(), "https://localhost:80"),
-        Arguments.of(aServerConfig().withTls(true).withPort(443).build(), "https://localhost"));
+        Arguments.of(aServerConfig().withTls(true).withPort(443).build(), "https://localhost"),
+        Arguments.of(aServerConfig().withNoContextPath().build(), "http://localhost:8000"),
+        Arguments.of(aServerConfig().withContextPath("auth").build(), "http://localhost:8000"),
+        Arguments.of(aServerConfig().withContextPath("/auth").build(), "http://localhost:8000"),
+        Arguments.of(aServerConfig().withContextPath("/context-path").build(), "http://localhost:8000"),
+        Arguments.of(aServerConfig().withContextPath("complex/context/path").build(), "http://localhost:8000")
```

Comment:
The test case that this argument list is used for is not affected by the context path, it's only affected by TLS and port. So I'd prefer not adding these extra cases.